src/background.js

```diff
@@ -79,7 +79,7 @@ chrome.runtime.onInstalled.addListener(async () => {
     } else {
       chrome.storage.local.set({
         options: {
-          announceMovement: false,
+          announceMovement: true,
```

Comment:
I think we should rename this to announceZoom, don't you?

Reply:
I think it may be beneficial to keep it as annoucemovement to align with web-map-custom-element. Also, since there is a high possibility that we expand beyond just announcing zoom, the name might be more reflective on the bigger picture (movement is just limited to zoom right now but will expand.